arch/posix/include/posix_arch_internal.h

```diff
@@ -9,13 +9,13 @@
 
 #include "toolchain.h"
 
-#define _SAFE_CALL(a) _safe_call(a, #a)
+#define Z_SAFE_CALL(a) z_safe_call(a, #a)
```

Comment:
Please rename this PC_SAFE_CALL and pc_safe_call respectively (this macro and function are meant to be private to the posix arch, and should not appear to be usable by anybody else)

Reply:
Fixed.